program/tests/duplicate_block_proof.rs

```diff
@@ -492,17 +626,24 @@ async fn improper_sigverify() {
         shred1: shred1.payload().as_ref(),
         shred2: shred2.payload().as_ref(),
     };
-    let data = duplicate_proof.pack();
+    let data = duplicate_proof.pack_proof();
 
     initialize_duplicate_proof_account(&mut context, &authority, &account).await;
     write_proof(&mut context, &authority, &account, &data).await;
 
     // Replace one of the signature verifications with a random message instead
     let message = Hash::new_unique().to_bytes();
     let signature = <[u8; SIGNATURE_BYTES]>::from(leader.sign_message(&message));
-    let mut instructions =
-        slashing_instructions(&account.pubkey(), slot, leader.pubkey(), &shred1, &shred2);
-    const MESSAGE_START: usize = 1 + 8 + 8 + 32;
+    let mut instructions = slashing_instructions(
+        &reporter,
+        &destination,
+        &account.pubkey(),
+        slot,
+        leader.pubkey(),
+        &shred1,
+        &shred2,
+    );
+    const MESSAGE_START: usize = 1 + 8 + 8 + 32 + 32;
```

Comment:
Sorry for smooth-brain question: This change is needed because of the added `destination` in the instruction data, correct?

Reply:
Yep (and now reporter), I just exposed a fn so that the test doesn't have to do this math